pkg/controller/container-runtime-config/helpers.go

```diff
@@ -156,6 +178,15 @@ func findRegistriesConfig(mc *mcfgv1.MachineConfig) (*igntypes.File, error) {
 	return nil, fmt.Errorf("could not find Registries Config")
 }
 
+func findPolicyJSON(mc *mcfgv1.MachineConfig) (*igntypes.File, error) {
```

Comment:
I can’t see any caller; was this intended to be added to `generateOriginalContainerRuntimeConfigs`?

Reply:
woops, yeah fixed.